bundles/org.openhab.ui/web/build/webpack.config.js

```diff
@@ -86,7 +86,11 @@ module.exports = {
           resolvePath('node_modules/ssr-window')
         ]
       },
-
+      {
+        test: /(blockly\/.*\.js)$/,
+        enforce: "pre",
+        use: ["source-map-loader"],
+      },
```

Comment:
```suggestion
      {
        test: /(blockly\/.*\.js)$/,
        enforce: "pre",
        use: (env === 'development' || buildSourceMaps) ? ["source-map-loader"] : [],
      },
```
I have to confirm this suggestion above works but I'd not include source maps in production builds in any case - unless the environment variable SOURCE_MAPS is set, then it would build the source maps for the entire UI, see lines 20 & 44 in these files. Source maps can be heavy in size and easily add megabytes to the openHAB distribution archive.
If `env === 'development'` (i.e. when you use `npm start` to start the UI with the development server) then it's fine to include any source maps.

Reply:
I totally agree not to have in production.